src/order/closure.lean

```diff
@@ -106,11 +106,35 @@ c.closure_top
 lemma closure_le_closed_iff_le {x y : α} (hy : c.closed y) : x ≤ y ↔ c x ≤ y :=
 by rw [← c.closure_eq_self_of_mem_closed hy, le_closure_iff]
 
-/-- The set of closed elements has a galois insertion to the underlying type. -/
+/-- The set of closed elements has a Galois insertion to the underlying type. -/
 def gi : galois_insertion c.to_closed coe :=
 { choice := λ x hx, ⟨x, le_antisymm hx (c.le_closure x)⟩,
   gc := λ x y, (c.closure_le_closed_iff_le y.2).symm,
   le_l_u := λ x, c.le_closure _,
   choice_eq := λ x hx, le_antisymm (c.le_closure x) hx }
 
 end closure_operator
+
+variables {α} (c : closure_operator α)
+
+/--
+Every Galois connection induces a closure operator given by the composition. This is the partial
+order version of the statement that every adjunction induces a monad.
+-/
+def galois_connection.closure_operator {β : Type u} [preorder β]
```

Comment:
should this be `@[simps]` or have some more simp-lemmas about it, apart from the one below?

Reply:
yup, added simps